apps/nginx/management/commands/https.py

```
@@ -191,6 +222,35 @@ def _get_or_create_certificate(
+            certbot.dns_credential
+            or DNSProviderCredential.objects.filter(
+                provider=DNSProviderCredential.Provider.GODADDY,
+                is_enabled=True,
+            )
+            .order_by("pk")
+            .first()
+        )
+        if credential is None:
+            raise CommandError(
+                "GoDaddy DNS validation requires an enabled DNS credential in admin (DNS > DNS Credentials)."
+            )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)